router.go

```diff
@@ -170,6 +170,15 @@ func (r *Router) Navigate(path string) {
 	}
 }
 
+// Can navigate returns true if the specified path can be navigated by the
```

Comment:
This is just nitpicking, but would could you change the first word to `CanNavigate` to match the name of the method exactly?

I like to follow the guidelines for documenting Go code, found here: https://blog.golang.org/godoc-documenting-go-code.